packages/uix-core/src/tunnel/tunnel.ts

```diff
@@ -81,6 +82,7 @@ export class Tunnel extends EventEmitter {
   private _messagePort: MessagePort;
 
   config: TunnelConfig;
+  isConnected: boolean;
```

Comment:
Adding connected state allows event handlers for `destroyed` to run timer cleanups conditionally.